src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Publish.targets

```diff
@@ -192,6 +192,12 @@ Copyright (c) .NET Foundation. All rights reserved.
         <RelativePath>@(_DebugSymbolsIntermediatePath->'%(Filename)%(Extension)')</RelativePath>
       </ResolvedFileToPublish>
 
+      <!-- Copy the documentation file (.xml), if any -->
+      <ResolvedFileToPublish Include="@(_DebugSymbolsIntermediatePath)"
```

Comment:
> There is a good use case for using documentation files at runtime in web projects.

Then it seems like this behavior should be added to Web projects, not all projects.

This is the advantage we get to using MSBuild - it is extremely extensible.  The core SDK doesn't need to do everything. It implements a core set of behaviors. And others can build on top of that.  Just because a certain class of projects need a behavior doesn't mean it needs to be in the core.